### PR TITLE
[ADD] sale_order_delivered_weight: New module to 12.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
Sale order delivered weight: New module to 12.0

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @manuelcalerosolis. Where is the module ``sale_order_weight`` on which your module depends ?

thanks.

**Comment by manuelcalerosolis** [comment]
Here:

https://github.com/OCA/sale-workflow/pull/1557

File: sale_order_delivered_weight/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    total_delivered_weight = fields.Float(
+        compute='_compute_total_delivered_weight',
+        string='Total Delivered Weight',
+        store=True
+    )
+
+    @api.depends('order_line.qty_delivered')
```
Comment (rousseldenis) [inline sale_order_delivered_weight/models/sale_order.py:16]: Do miss "order_line.product_id.weight" ?

Comment (manuelcalerosolis) [reply sale_order_delivered_weight/models/sale_order.py:16]: @rousseldenis this module has been replaced by :

https://github.com/OCA/sale-workflow/pull/1557

**Comment by legalsylvain** [comment]
this PR should be closed, as https://github.com/OCA/sale-workflow/pull/1557 supersed it.

**Pull request closed without merging**